ec/ecresource/deploymentresource/schema.go

```diff
@@ -25,44 +25,52 @@ import (
 func NewSchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"version": {
-			Type:     schema.TypeString,
-			Required: true,
-		},
-		"name": {
-			Type:     schema.TypeString,
-			Optional: true,
+			Type:        schema.TypeString,
+			Description: "Required Elastic Stack version to use for all of the deployment resources",
```

Comment:
I noticed none of these descriptions have a period at the end, is this intentional?

Also, same comments for all descriptions as the ones I left above :)

Reply:
I don't think so! My aim was to have these without any periods in the source `.go` files